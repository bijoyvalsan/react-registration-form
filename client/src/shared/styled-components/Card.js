import styled from 'styled-components';

export const Card = styled.div`
    background-color: #fff;
    box-shadow: 0px 1px 3px 0px rgba(0,0,0,0.2), 0px 1px 1px 0px rgba(0,0,0,0.14), 0px 2px 1px -1px rgba(0,0,0,0.12);
    border-radius: 4px;
    overflow: hidden;
    height: auto;
    max-width: 600px;
    min-width: 600px;
    margin: auto;
`;