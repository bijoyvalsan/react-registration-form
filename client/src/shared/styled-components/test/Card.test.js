import React from 'react';
import 'jest-styled-components';
import renderer from 'react-test-renderer';

import { Card } from '../Card';

describe('Styled Components: Card', () => {
  it('renders correctly - Card Style Component', () => {
    const tree = renderer.create(<Card>Testing Card</Card>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
