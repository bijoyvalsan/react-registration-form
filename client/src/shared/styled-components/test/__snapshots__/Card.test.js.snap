// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled Components: Card renders correctly - Card Style Component 1`] = `
.c0 {
  background-color: #fff;
  box-shadow: 0px 1px 3px 0px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 2px 1px -1px rgba(0,0,0,0.12);
  border-radius: 4px;
  overflow: hidden;
  height: auto;
  max-width: 600px;
  min-width: 600px;
  margin: auto;
}

<div
  className="c0"
>
  Testing Card
</div>
`;
